cache_images/podman_tooling.sh

```diff
@@ -1,20 +1,34 @@
 
-# This script is sourced from *_packaging.sh script to install common/shared
+# This script is executed from *_packaging.sh script to install common/shared
 # tooling from the containers/podman repository.  It expects
-# a go 1.13+ environment has already been setup.  The script should
-# not be used for any other purpose or from any other context.
+# a go 1.16+ environment has already been setup, and should be called
+# as root or through sudo.  The script should not be used for any other
+# purpose or from any other context.
+
+set -e
+
+SCRIPT_FILEPATH=$(realpath "${BASH_SOURCE[0]}")
+SCRIPT_DIRPATH=$(dirname "$SCRIPT_FILEPATH")
+REPO_DIRPATH=$(realpath "$SCRIPT_DIRPATH/../")
+# shellcheck source=./lib.sh
+source "$REPO_DIRPATH/lib.sh"
+
+echo "Configuring Go environment"
+export GOPATH="${GOPATH:-/var/tmp/go}"
+mkdir -p "$GOPATH"
+export GOSRC=${GOPATH}/src/github.com/containers/podman
+export GOCACHE="${GOCACHE:-$GOPATH/cache}"
+eval $(go env | tee /dev/stderr)
+export PATH="$GOPATH/bin:$PATH"
 
 echo "Installing runtime tooling"
-export GOPATH="${GOPATH:/var/tmp/go}"
-export GOSRC=/var/tmp/go/src/github.com/containers/podman
-export GOCACHE="${GOCACHE:-/root/.cache/go-build}"
 lilto git clone --quiet https://github.com/containers/podman.git "$GOSRC"
 
 cd "$GOSRC" || die "Podman repo. not cloned to expected directory: '$GOSRC'"
 # Calling script already loaded lib.sh
 # shellcheck disable=SC2154
```

Comment:
Is this `shellcheck` still needed? (I see the same 2154 number below, next to a variable, so I'm guessing 2154 has to do with variables)